docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+        type: 'string',
+        description: 'The header for the component',
+        required: true,
+      },
+      {
+        name: 'value',
+        type: 'string',
+        description: 'The main datapoint value (e.g., 1.23M)',
+        required: true,
+      },
+      {
+        name: 'trend',
+        type: '{| accessibilityLabel: string, value: string |}',
+        description: `Object detailing the trend value (change in time - e.g., +30%), and accessibilityLabel to describe the trend's icon (e.g., "Trending up")`,
+      },
+      {
+        name: 'trendSignal',
+        type: `'good' | 'bad' | 'neutral' | 'auto'`,
+        defaultValue: 'auto',
+        description: `A visual indicator whether the trend is considered "good", "bad" or "neutral". By setting \`trendSignal\` to \`auto\`, a positive trend will be considered "good", a negative trend will be considered "bad" and a trend of zero will be considered "neutral"`,
+      },
+      {
+        name: 'infoText',
+        type: 'string',
+        description: 'Contextual information displayed in a tooltip to describe the Datapoint',
+      },
+      {
+        name: 'size',
+        type: `'md' | 'lg'`,
+        defaultValue: 'md',
+        description: 'Used to set the size of the datapoint',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`infoText\` and \`trendAccessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection columns={2} title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="size=md"
+        defaultCode={`
+        <Datapoint size="md" title="Spend" value="$1.23M" />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="size=lg"
+        defaultCode={`
+        <Datapoint size="lg" title="Spend" value="$1.23M" />
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`Use \`trend\` to display the delta in the value of a Datapoint over time. Make sure to provide an \`accessibilityLabel\` when the trend is above or below zero.`}
+      title="Trend"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Flex direction="column" gap={4}>
+  <Datapoint title="Pin clicks" value="1.23k" trend={{value: 12, accessibilityLabel: "Trending up"}} />
+  <Datapoint title="Saves" value="123" trend={{value: 0, accessibilityLabel: ""}} />
+  <Datapoint title="Total impressions" value="1.23M" trend={{value: -5, accessibilityLabel: "Trending down"}}  />
+</Flex>
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`Use \`trendSignal\` to explicitly the whether the \`trend\` is considered "good", "bad" or "neutral". By default, a positive trend will be considered "good", a negative trend "bad" and a trend of 0 "neutral".`}
```

Comment:
typo: "to explicitly set whether the..."

Reply:
I'd also consider switching the order of these two sentences:

By default, a positive trend will be considered "good", a negative trend "bad" and a trend of 0 "neutral". However, the \`trendSignal\` property can be used to explicitly set whether the \`trend\` is considered "good", "bad" or "neutral", as demonstrated below.

Reply:
Love it. Done!